docs/tools-reference/custom-kernels-distros/custom-compiled-kernel-debian-ubuntu.md

```diff
@@ -12,6 +12,8 @@ published: ''
 title: 'Custom Compiled Kernel on Debian & Ubuntu'
 ---
```

Comment:
Similar to its cousin, this doc could use a 155 character or less metadescription describing what will be accomplished by following this guide.